src/main/java/somethingrandom/interfaceadapters/additem/AddItemView.java

```diff
@@ -0,0 +1,143 @@
+package somethingrandom.interfaceadapters.additem;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.event.KeyEvent;
+import java.awt.event.KeyListener;
+import java.beans.PropertyChangeEvent;
+import java.beans.PropertyChangeListener;
+
+public class AddItemView extends JPanel implements ActionListener, PropertyChangeListener {
+
+    private static JPanel panel = new JPanel();
+    private static JFrame frame = new JFrame();
+    private static JLabel itemTitle = new JLabel();
+    private static JTextField itemtitleInput = new JTextField(100);
```

Comment:
(a) should probably be `itemTitleInput`
(b) if you want, we can probably go past 100 characters, but it's not important.

Reply:
You actually make new ones in `main`, so you should probably just make them here or just make them there.